src/nft/components/details/detailsV2/NftDetails.tsx

```diff
@@ -8,11 +9,29 @@ interface NftDetailsProps {
   collection: CollectionInfoForAsset
 }
 
+const DetailsBackground = styled.div<{ backgroundImage: string }>`
+  position: fixed;
+  top: ${({ theme }) => `-${theme.navHeight}px`};
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: ${({ backgroundImage }) => `url(${backgroundImage})`};
+  filter: blur(100px);
+  opacity: ${({ theme }) => (theme.darkMode ? 0.2 : 0.24)};
+`
+
+const DetailsContentContainer = styled.div`
+  z-index: 2;
```

Comment:
we have a Z_INDEX constant defined - could you use that?

Reply:
My understanding is that we use bootstraps z-index rules: https://getbootstrap.com/docs/5.0/layout/z-index/ - their rules say to use 1,2, or 3 for handling z-indexes inside of components. Is that not appropriate to apply here. All of the z-indexes we have in themes are large

Reply:
oh i guess you are right. idk what do you think about just still making them constants in there? it says `1, 2, and 3 for default, hover, and active states` so maybe naming them something like that?

Reply:
I like that